ui/App/UserProfileScreen.svelte

```diff
@@ -45,9 +71,20 @@
         {urn} />
     </Header>
 
-    <ActionBar>
-      <TabBar slot="left" {tabs} />
-    </ActionBar>
-    <ProjectsTab {urn} />
+    {#if $store.status === remote.Status.Success}
+      {#if (Array.isArray($store.data) && $store.data.length === 0) || $store.data === null}
```

Comment:
If `$store.data` is present, then it is always an array so the following should be sufficient.

```suggestion
      {#if $store.data.length === 0}
```

Reply:
Typescript complains if you do that. It says that `data` isn't an array. I guess that's why you have: `{#if Array.isArray($store.data) && $store.data.length === 0`

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2398/commits/485ac38bc0f582ae01cf121375c22e1245297692